pkgs/development/python-modules/nbclassic/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, notebook
+, pythonOlder
+, jupyter_server
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "nbclassic";
+  version = "0.2.5";
+  disabled = pythonOlder "3.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1as08rdvivsjgi7q65c3mx55c34s7jlqxks25zk3v9knmcb23np6";
+  };
+
+  checkInputs = [ pytest ];
+  propagatedBuildInputs = [ jupyter_server notebook ];
+
+  doCheck = false;
```

Comment:
Why are tests disabled if there is a checkPhase?

Also if they are disabled please add a pythonImportsCheck.